pkg/apis/scorecard/v1alpha3/register.go

```diff
@@ -0,0 +1,14 @@
+package v1alpha3
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// SchemeGroupVersion is group version used to register these objects
+	SchemeGroupVersion = schema.GroupVersion{Group: "scorecard.operatorframework.io", Version: "v1alpha3"}
```

Comment:
If we're removing `SchemeBuilder` and really have no need to integrate with schemes, should be rename this to simply `GroupVersion`?